docs/api/websocket.md

```diff
@@ -1,9 +1,21 @@
 ---
 title: "WebSocket API"
 ---
-
 Home Assistant contains a WebSocket API. This API can be used to stream information from a Home Assistant instance to any client that implements WebSockets. We maintain a [JavaScript library](https://github.com/home-assistant/home-assistant-js-websocket) which we use in our frontend.
 
+
+## Endpoint Address
+
+You will need a valid access token.  Typically a long lived access token is used for API reqeusts.  
+Generate an access token via the [user profile page](https://www.home-assistant.io/docs/authentication/#user-accounts) in the front end, or via the [auth api](https://developers.home-assistant.io/docs/auth_api/#long-lived-access-token)
+
+The default port number is 8123.  If you change  the HTTP port in configuration.yaml, use the port defined in the config.  
+Connect your websocket implementation to ws://localhost:[PORT NUMBER]/api/websocket.  
```

Comment:
localhost might be confusing in this case. Also, the port can be different (not just because of the configuration.yaml, but because of proxies or the use of Home Assistant Remote UI).
Maybe just say `/api/websocket` on the web address of the Home Assistant instance?